docket_scrap/spiders/docket_spider.py

```diff
@@ -0,0 +1,213 @@
+"""
+Scrapy Spider that crawls RIPUC webpage for commission dockets
+run by command "scrapy crawl docket" on cmd
+"""
+
+from w3lib.html import remove_tags
+import re
+import scrapy
+from urllib.parse import urljoin
+import csv
+
+from ..items import Docket
+
+# from scrapy.linkextractors import LinkExtractor
+# from scrapy.http.request import Request
+
+
+class DocketSpider(scrapy.Spider):
+    # name of the spider
+    name = 'docket'
+
+    # url of the target page
+    start_urls = [
+        'http://www.ripuc.ri.gov/eventsactions/docket.html'
+    ]
+
+    # base url of the website category
+    base_url = 'http://www.ripuc.ri.gov/eventsactions/'
+
+    def parse(self, response):
+        """
+        Handling the initial response of the webpage
+        """
+
+        # Extracting the menu body of the webpage
+        body = response.css("table")[2]
+
+        # Extracting the table of the dockets
+        docket_table = body.css("table")[1]
+
+        # Importing all rows
+        rows = docket_table.xpath('tr')
+
+        # temporary storage for dockets for extraction purposes
+        temporary_storage = dict()
+
+        # Looping through each row for data
```

Comment:
1. Do not add unnecessary new lines
2. You should not add in-line comments unless necessary/context is not clear without the comments. The variable names should convey the meaning properly. e.g. temporary_storage does not indicate what items will be stored in it.